### DISCUSSION TITLE
Allowed Amount File - Including Prescription Drugs

### DISCUSSION THREAD

**Discussion by AJLgeHG** [CMSgov/price-transparency-guide#420] [General]
For including prescription drugs data on the Allowed Amounts machine readable file, should that data be exclusively for a prescription drug benefit? 

Or, should it be allowed amounts and historical claims paid for prescription drugs that are a part of members' medical benefit? 

Or, should it be both?

**Comment by AJLgeHG** [comment]
@shaselton-usds   Hi. My thought is, the Pharmacy Benefit and Drug Cost Report will capture prescription cost data for Prescription Drugs, so for the Allowed Amounts file to include prescription drugs data that would be prescription drugs data that is billed to medical claims and not that which is solely for a Prescription Drug plan or Prescription Drug benefit.

Is that correct? Could you please confirm?

**Reply by RLTx1391** [reply]
I would think it's just for those on the medical benefit. The Prescription Drug file was delayed pending rulemaking and they called out the potential duplication with the RxDC (204) report, so it doesn't seem including both would be of value.

**Comment by shaselton-usds** [comment]
@AJLgeHG -- thanks for the questions. 

We're trying to understand this a bit better. Would it be possible a bit more context, maybe a specific example? 

We are trying to understand the context of the situation in related to the Allowed Amounts file vs the Rx file (which was delayed as @RLTx1391 pointed out -- the Rx file was intended to somewhat mirror the in-network file in terms of contract data being disclosed; quite different than the allowed amount file).

**Reply by AJLgeHG** [reply]
Hello @shaselton-usds  - Thanks for following up. Initially when the rules were published for the Allowed Amounts file (when the requirement was for 3 machine-readable files where now it's 2 - In-Network and Allowed Amount) the requirements had said we were to include historical data for prescription drugs, so out-of-network drug pricing needed to be included.  The rules have since changed as the information was thought to be duplicative because a Prescription Drug Report (RxDC) was also required. 

In the "Definition" column of the "description" field for the "Out-of-Network" object on the Allowed Amounts schema, the text reads "In the case of NDCs for prescription drugs, the plain language description must be the proprietary and nonproprietary names assigned to the NDC by the FDA."  This indicates that historical data for prescription drugs should be included/is expected. 

As we also need to report Prescription Drug data on the RxDC Report, I am not sure if the expected prescription drug reporting on the Allowed Amounts machine-readable file should be for prescription drug provided in a medical setting (claims on medical plans) or if they should be those which are claims on prescription drug plans. 

My thought is it should be those associated with or tied to a medical benefit or medical plans. It is also possible that prescription drug data should not be provided on the Allowed Amounts at all. However, I'm not sure. Could you advise?

Thanks for help.

**Reply by RLTx1391** [reply]
Agreed- this is how we were interpreting it- reporting only allowed amounts paid under the medical benefit.

**Reply by AJLgeHG** [reply]
@shaselton-usds  - Hi. Is the interpretation here correct? Please advise.

Thanks.

**Reply by RLTx1391** [reply]
@shaselton-usds Just bumping this again.
